Classes/PassiveTreeView.lua

```diff
@@ -280,6 +280,19 @@ function PassiveTreeViewClass:Draw(build, viewPort, inputEvents)
 				build.itemsTab:SelectControl(slot)
 				build.viewMode = "ITEMS"
 			end
+		--[[ Only allow node editing in these situations:
+				Vaal (Glorious Vanity): 		any non-keystone
+				Maraketh (Brutal Restraint): 	only notables, +dex already set
```

Comment:
Indentation is off.

Reply:
Looks alright in my IDE, putting it back one tab would leave it to the left of the first line:

![image](https://user-images.githubusercontent.com/1209372/94294497-89438d80-ff25-11ea-8b0e-a7496d13c091.png)

vs.

![image](https://user-images.githubusercontent.com/1209372/94294583-a4ae9880-ff25-11ea-8af2-aa4fa172126d.png)